### PR TITLE
feat(article): forward silhouette from article response

### PR DESCRIPTION
**Pull request by nongio**
related to issue 2004

### CODE REVIEW

**Comment by MrDzi** [comment]
👍

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "atlas-sdk-core",
-  "version": "4.13.0",
+  "version": "4.14.0",
```
Comment (hm6293) [inline package.json]: This is gonna be `version: 4.16.0` since we are currently on `version: 4.15.0`

**Comment by Etchs** [comment]
👍

**Comment by nongio** [comment]
👍

**Pull request merged**